tests/test_mtdome_csc.py

```diff
@@ -653,5 +691,87 @@ async def test_status_error(self):
                 faultCode=expected_fault_code,
             )
 
+    async def test_exitFault(self):
+        async with self.make_csc(
+            initial_state=salobj.State.STANDBY,
+            config_dir=None,
+            simulation_mode=1,
+        ):
+            await self.set_csc_to_enabled()
+
+            # Prepare the lower level components
+            self.csc.mock_ctrl.amcs.status = MotionState.FAULT
+            self.csc.mock_ctrl.apscs.status = MotionState.FAULT
+            self.csc.mock_ctrl.lcs.status[:] = MotionState.FAULT.name
```

Comment:
Why is this line different? Is that difference necessary?

Reply:
The statuses are passed on as strings, so it is necessary.